packages/components/src/circular-progress-bar/README.md

```diff
@@ -0,0 +1,22 @@
+# Circular Progress Bar
+
+This component is used to display a circular progress bar as part of the cohesive Launchpad Experience
+See https://cylonp2.wordpress.com/2022/12/14/a-cohesive-launchpad-experience/
+It displays a gray circle and another blue circle on top of it showing the progress.
+It renders text in the center in the center: X/Y where X is the current step and Y is the total number of steps
+The currentStep and numberOfSteps props are used to calculate represent the current progress as a fraction
+
```

Comment:
I wonder if we should name this something like "LaunchPad Circular Progress Bar", or put a big warning that this is intended for launchpad checklists (in their various places) and not necessarily for other unrelated progress bar solutions. I say that since our launchpad/checklist icons used for distinguishing and returning will all use this, and if other interfaces start using the circular progress bar format as well, that could get confusing.

I see you already noted "This component is used to display a circular progress bar as part of the cohesive Launchpad Experience", but maybe good to add that its not intended for non-launchpad related interfaces.